seleniumJavaExperiment/src/test/java/SeleniumTests.java

```diff
@@ -0,0 +1,86 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pageObjects.ABtestPage;
+import pageObjects.CheckboxesPage;
+import pageObjects.DropDownPage;
+import pageObjects.HomePage;
+
+import java.io.PrintStream;
+
+public class SeleniumTests {
+
+    WebDriver driver;
+
+    HomePage homePage;
+    ABtestPage abTestPage;
+    CheckboxesPage checkboxesPage;
+    DropDownPage dropdownpage;
+
+    @BeforeClass
+    public void beforeClass () {
+        System.setProperty("webdriver.chrome.driver", "./src/test/resources/chromedriver/chromedriver.exe");
```

Comment:
Please make use of webdriver binaries - 
[Automatically download WebDriver Binaries](https://kenilpfadia.medium.com/automatically-download-webdriver-binaries-dd7ecc77ae55)

Let me know if you need any help here.

Reply:
@kenil-fadia I need your help.